lib/ex_machina.ex

```diff
@@ -58,6 +58,17 @@ defmodule ExMachina do
       def create_list(number_of_factories, factory_name, attrs \\ %{}) do
         ExMachina.create_list(__MODULE__, number_of_factories, factory_name, attrs)
       end
+
+      def factory(factory_name, attrs \\ [])
```

Comment:
I'm a bit confused what are the two function heads? I thought since we have the macro it'll always be `factory(name, attrs)`?